src/Tribe/Rewrite.php

```diff
@@ -581,8 +588,11 @@ static function ( $rule_query_string ) use ( $pattern ) {
 		 * @param array $our_rules An array of rewrite rules handled by our code, in the shape
 		 *                         `[ <rewrite_rule_regex_pattern> => <query_string> ]`.
 		 *                         E.g. `[ '(?:events)/(?:list)/?$' => 'index.php?post_type=tribe_events&eventDisplay=list' ]`.
+		 * @param array<string,string> All the current rewrite rules, before any filtering is applied; these have the
+		 *                             same `<pattern => rewrite >` format as the previous argument, which is the
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 113 characters

<sub>hash: ffb381ec31012339855bbf9b35e8ca1c</sub>